apps/aot/src/app/about/page.tsx

```diff
@@ -94,17 +92,17 @@ export default function AboutPage() {
             >
               TypeScript Playground
             </a>{' '}
-            preferred over our editor(at least for now 🤐). Just copy the code from the test
-            section, start coding, and when you nail it, pop your code (except the tests) back into
-            our editor.
+            preferred over our editor (at least for now 🤐). You can copy the code from the test
+            section into the playground, start coding, and when you nail it, pop your code (except
+            any tests) back into our editor.
           </li>
           <li>
             The test cases are your friends! They’ll guide you through what we’re asking for. When
             things get trickier, these cases will be your trusty helpers.
           </li>
           <li>
-            Join the advent-of-typescript channel in our discord linked above just for chewing over
-            these challenges. Jump in, chat, and share your thoughts on solutions. We highly
+            Join the advent-of-typescript channel in our discord linked earlier just for chewing
```

Comment:
> above

Accessibility side note: it's not "above" if you're not visually lookinga t the page.